src/styles/blocks/card.scss

```diff
@@ -0,0 +1,93 @@
+.card {
+  box-sizing: border-box;
+  padding: 32px 16px 15px;
+  width: 200px;
+  border-radius: 5px;
+  background-color: #fff;
+  color: $color-main;
+  border: 1px solid #f3f3f3;
+
+  &__image {
+    padding-inline: 3px;
+    margin-bottom: 36px;
+    & img {
+      width: 100%;
+      height: 134px;
+      background-size: cover;
```

Comment:
The `background-size: cover;` property should be applied to a background image. Since this is an `img` tag, you should use `object-fit` instead to define how the content fits in the box.